### DISCUSSION TITLE
dumi2的移动端主题容器二维码预览功能

### DISCUSSION THREAD

**Discussion by leegsen7** [umijs/dumi#1688] [Q&A]
默认主题是不提供这个功能了吗，我没找到对应的配置

**Comment by PeachScript** [comment]
应该是漏掉了，欢迎提交 PR 完善，组件位置：https://github.com/umijs/dumi/blob/b5bb4e1b98733fb36d75a290e54c3184ee3fdd52/suites/theme-mobile/src/slots/PreviewerActions/index.tsx#L4

**Comment by PeachScript** [comment]
v2.2.1 已发布，包含该功能，感谢贡献
